app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,208 @@
+
+In this guide, you will set up two developer teams (more info about the scenario) and then enable Portal RBAC using a hypothetical scenario.
+
+## Prerequisites
+* Two [test developer accounts registered](/konnect/dev-portal/dev-reg/)
+* A [runtime configured](/konnect/getting-started/configure-runtime)
+
+## Configure developer teams 
+
+In this scenario, you are a product manager at a pizza company who is in charge of their online app. You're asked to create a Dev Portal that allows delivery companies to use their APIs to deliver the company's pizzas. However, you want to make sure only trusted delivery partners can build apps using your APIs and deliver your pizzas. 
+
+You've decided to create two groups of developers with different access levels to your APIs: 
+* Delivery Partners: This developer team will be able to view and consume your APIs so they can use them in their own apps to deliver your pizzas.
+* New Partners: This developer team will contain delivery groups that the company is still in the process of vetting and you don't want to allow them to consume your APIs in their apps yet. You give this group view-only privileges so they can see what your API specs look like. 
+
+{:.important}
+> **Important:** If you currently have developers in your {{site.konnect_short_name}}, you must configure developer teams before enabling Portal RBAC (which is disabled by default). If you enable Portal RBAC before configuring teams for your developers, it will prevent all developers from seeing any services in your Dev Portal. We recommend setting up your teams and permissions before enabling RBAC which will allow for a seamless transition: developers see what they're supposed to, instead of nothing at all.
+
+### Create an API Product
+
+{% navtabs %}
+{% navtab Konnect UI %}
+In {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.com/us/runtime-manager), select the **Default** runtime group and follow these steps:
+
+1. Select **Gateway Services** from the side navigation bar, then **New Gateway Service**.
+
+1. From the **Add a Gateway Service** dialog, enter the following to create a new service:
+    * **Name:** classic_pizzas
+    * **Upstream URL:** `http://mockbin.org`
+    * Use the defaults for the remaining fields.
+1. Click **Save**. 
+1. To create an API product for your service, navigate to **API Products** in the sidebar, click **Add API Product** and enter `Classic Pizzas` in the **Product Name** field.
+1. Click **New Version** from the Actions dropdown menu and enter `v1` in the **Version Name** field and then click **Create**.
+1. Click the **v1** version and then click **Link** to add a service.
+1. Select **default** from the **Select Runtime Group** dropdown menu, select **classic_pizzas** from the **Gateway Service** dropdown menu, and then click **Save**.
+
+{% endnavtab %}
+{% navtab API %}
+1. Create the service?
```

Comment:
I think this needs to be flushed out.

Reply:
❗ For reviewers: I couldn't find any Konnect API specs for this, should I just use the gateway API?

Reply:
That link is the konnect API spec

Reply:
@cloudjumpercat , you'll want the Portal RBAC spec https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696#/Services/create-service